src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="big-search-bar form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <button
+        class="button"
+        type="submit"
+        name="submit"
+      ></button>
```

Comment:
Why do you need this button? It does nothing. You should remove it.

Reply:
Yeasterday two tests on BackstopJS Report were failed, but on gitHub everything was correct, today BackstopJS Report showed me this results, but something wrong on gitHub. So what should I do? I changed .small-input top and bottom paddings back and forth. ![image](https://user-images.githubusercontent.com/105418918/213020744-2362472a-16cb-48c4-a57c-efac7a4d3614.png)